src/Components/OATModelList/OATModelList.tsx

```diff
@@ -103,10 +106,13 @@ const OATModelList = ({
                 type: SET_OAT_PROPERTY_EDITOR_MODEL,
                 payload: null
             });
+
             dispatch({
-                type: SET_OAT_DELETED_MODEL_ID,
-                payload: id
+                type: SET_OAT_CONFIRM_DELETE_TYPE,
+                payload: SET_OAT_DELETED_MODEL_ID
             });
+            dispatch({ type: SET_OAT_CONFIRM_DELETE_PAYLOAD, payload: id });
+            dispatch({ type: SET_OAT_CONFIRM_DELETE_OPEN, payload: true });
```

Comment:
Rather than having to dispatch three times, can we please dispatch once with an object with all of the required properties?